stp_zmq/test/test_zstack.py

```diff
@@ -165,3 +166,38 @@ def create_stack(name, handler=None):
             .format(len(expected_messages),
                     len(received_messages),
                     received_messages[-1])
+
+
+def testZStackCannotSendHugeData(tdir, looper, tconf):
+    names = ['Alpha', 'Beta']
+    (alpha, beta), _ = create_and_prep_stacks(names, tdir, looper, tconf)
+    nn = 'greetings' * int(tconf.MSG_LEN_LIMIT / 2)
```

Comment:
Can we create more specific tests for msg length:
- create a message of the length equal to MSG_LEN_LIMIT; check that the message can be send/received
- create a message of the length equal to MSG_LEN_LIMIT+1; check that the message is not send/received